specification/redis/resource-manager/Microsoft.Cache/stable/2020-06-01/redis.json

```diff
@@ -2071,6 +2071,18 @@
           "$ref": "#/definitions/RedisCreateProperties",
           "description": "Redis cache properties."
         },
+        "redisVersion": {
+          "type": "string",
+          "description": "Redis version. Valid values: (4, 6)",
```

Comment:
There shouldn't be any changes to existing stable api-versions.  This contract change should go in a new api-version.

Reply:
This new property is optional, so will not break any existing calls. Please let me know why this needs to be in a new api-version.

Swagger BreakingChange has also passed and guidelines also doesnt mention it as breaking change: https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/Breaking%20changes%20guidelines.md

Reply:
This is a bit of a gray area but if the tools aren't flagging you then I'll leave it up to the tools